src/classpath-openjdk.cpp

```diff
@@ -600,10 +598,24 @@ class MyClasspath : public Classpath {
         PROTECT(t, e);
```

Comment:
We can remove this `PROTECT` line since the local variable `e` is not used after the call to runRaw.